examples/nlp/machine_translation/create_tarred_dataset.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import pickle
+import time
+from pathlib import Path
+import tempfile
+import tarfile
+import numpy as np
+import youtokentome as yttm
+import math
+import json
+
+from nemo.collections.nlp.data import TranslationDataset
+from nemo.collections.nlp.modules.common.tokenizer_utils import get_tokenizer
+
+def create_pickled_dataset(args, src_fname, tgt_fname, num_tokens, encoder_tokenizer, decoder_tokenizer, tar_file_ctr, global_batch_ctr):
```

Comment:
Let's add doc strings to functions and classes.

Reply:
Done